client/app/collections/search.coffee

```diff
@@ -0,0 +1,35 @@
+{Filtered}  = BackboneProjections
+
+
+module.exports = class Search extends Filtered
+    constructor: (underlying, options = {}) ->
+        app     = require 'application'
+        @search = app.model.get 'filter'
+
+        options.filter = @filter.bind @
```

Comment:
We usually prefer to use the => instead of manual .bind
(in this case, at filter function definition L18). 

Any reason why you chose to use .bind instead ?